lib/data-service.js

```diff
@@ -27,6 +27,10 @@ class DataService extends EventEmitter {
       .on('topologyDescriptionChanged', (evt) => this.emit('topologyDescriptionChanged', evt));
   }
 
+  getConnectionParams() {
```

Comment:
How connectionParams are different from connectionOptions?

Reply:
Are you referring to something specific in `data-service` when you say `connectionOptions`? I'm not sure what they are